nixos/modules/security/acme.nix

```diff
@@ -139,6 +139,19 @@ in
         '';
       };
 
+      production = mkOption {
+        type = types.bool;
+        default = false;
```

Comment:
This at least needs to default to true, we can't just break everyone's LE installs.